src/Unosquare.RaspberryIO.Playground/System/SystemCamera.cs

```diff
@@ -43,9 +43,9 @@ private static void CaptureImage()
         {
             Console.Clear();
 
-            var imageWidth = Terminal.ReadNumber("Set the image width:", 640);
-            var imageHeight = Terminal.ReadNumber("Set the image height:", 480);
-            var fileName = Terminal.ReadLine("Set the file name:");
+            var imageWidth = 0;
```

Comment:
you need to read values or set defaults.

Reply:
This change is crap, I didn't intend to include this :-(